lib/tasks/support/teaching-internship-providers.md.erb

```diff
@@ -5,8 +5,6 @@ subcategory: Postgraduate teacher training
 description: |-
   Find paid teaching internships to gain new skills and see what classroom life is like. Explore chemistry, computing, languages, maths and physics internships.
 image: false
-promo_content:
```

Comment:
Does the cta_adviser need adding here too (you've added it to the AO page `lib/task`)?

One for @martyn-w to advise on!

Reply:
@emmaachesongray @gemmadallmandfe my reading of this PR is the new adviser promo _will_ be displayed as it has been added to `app/views/content/train-to-be-a-teacher/teaching-internship-providers/_listing.html.erb` - and so does not need to be added to this file.